common/v2/services/WalletService/deterministic/ledger.ts

```diff
@@ -45,6 +45,8 @@ export class LedgerWallet extends HardwareWallet {
   }
 
   public async signRawTransaction(t: EthTx): Promise<Buffer> {
+    // Disable EIP155 in Ethereumjs-tx since it conflicts with Ledger
+    t = new EthTx(t, { chain: t.getChainId(), hardfork: 'tangerineWhistle' });
```

Comment:
For best practices sake please avoid reassigning a functions arguments.
https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/

Since objects are passed by reference in JS, the previous implementation of `signRawTransaction` which modifies an arguments params is particularly dangerous and cause some nasty bugs.

Reply:
Wasn't aware, thanks 😄